src/control/cad.js

```diff
@@ -623,6 +634,38 @@ class CadControl extends Control {
     const snapCoords = [];
     const snapCoordsAfter = []; // store the direct next point in the coordinate array
 
+    // Parse the list of coords, we will add the ortho and segment lines which
+    // are closest to the users cursor up until the limit of `this.limitGuidesPerFeature`
+    const parseCoordList = (coords) => {
+      let includedCoords;
+      if (coords.length > this.limitGuidesPerFeature) {
+        includedCoords = [];
+        // Check the euclidean distance between coord at index and the
+        // included vertices so far. Create an array of `this.limitGuidesPerFeature`
+        // closest items.
+        for (let index = 0; index < coords.length; index += 1) {
+          const addIndex = sortedIndexBy(includedCoords, index, (i) =>
+            euclideanDistance(coords[i], coordinate),
+          );
+          if (addIndex < this.limitGuidesPerFeature) {
+            includedCoords.splice(
+              addIndex,
+              includedCoords.length === this.limitGuidesPerFeature ? 1 : 0,
+              index,
+            );
+          }
+        }
+      } else {
+        includedCoords = times(coords.length);
```

Comment:
We don't want to rely  on lodash speciially when native function can do the job. It seems Array.from(Array(coords.length))  does the same .